assets/js/vue/components/research-group/DatasetsTab.vue

```diff
@@ -1,25 +1,15 @@
 <template>
     <b-card-text class="vh-100">
-        <b-card class="card-product" v-for="dataset in datasets" v-bind:key="dataset.id">
-            <b-link class="dataset-row" target="_blank" :href="getDatalandRoute(dataset.udi)">
-                <b-card-body>
-                    <div class="row">
-                        <article class="col-lg-12">
-                            <b-card-text>
-                                {{ dataset.citation }}
-                            </b-card-text>
-                        </article>
-                    </div>
-                </b-card-body>
-            </b-link>
-        </b-card>
+        <ResultRow :resultRowData="dataset" v-for="dataset in datasets" v-bind:key="dataset.udi"/>
```

Comment:
Do we want to change the name of this Vue component to DatasetRow in this Pull request? Now, that we are reusing Vue components I thought DatasetRow would make more sense

Reply:
I think we should, if possible, otherwise we might never get around to it later.